lib/kernel/src/code.erl

```diff
@@ -109,108 +109,20 @@ selectively.
 
 > #### Change {: .info }
 >
-> The existing experimental support for archive files will be changed
-> in a future release. As of Erlang/OTP 27, the function `code:lib_dir/2`,
-> the `-code_path_choice` flag, and using `m:erl_prim_loader` for
-> reading files from an archive are deprecated.
+> Support for archive files is an experimental feature, and as such can be
+> subject to incompatible changes.
 >
-> `escript` scripts that use archive files should use
+> In Erlang/OTP 28, the archive support in `m:erl_prim_loader` and the
+> the code server has been removed. Archives can be still be used for
+> escripts, but the internal implementation of archives for escripts
```

Comment:
What does "internal implementation" mean here? The file format, or the code in the BEAM. If the latter, that's utterly irrelevant for users of escripts. If the former, it should be documented. (Haven't read the rest of the PR.)

Reply:
It means that instead of storing a zip archive in the escript, the BEAM files and the other files are stored in a different way. We intentionally didn't document how exactly they are stored, because we want to force tools that handle escripts containing multiple files to use `escript:create/2` and `escript:extract/2`.